portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,30 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.gson.Gson;
```

Comment:
fix the order here

Reply:
I was not entirely sure what order I should have so I took my best guess: a1fda42.

Reply:
https://google.github.io/styleguide/javaguide.html#s3.3-import-statements

Reply:
Okay that is what I thought it would be. I will make sure to skim over the style guides tomorrow for JS and Java.